spec/cassettes/ActiveCampaign_API_Fields/_update_field/returns_a_user_hash.yml

```diff
@@ -0,0 +1,210 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: "<API_ENDPOINT>/fields"
+    body:
+      encoding: UTF-8
+      string: '{"field":{"title":"Custom text - updated","type":"text","descript":"A
```

Comment:
As you can see here the creation is already doing this with the updated field. If you have a look at my previous tests you can see how it needs to be wired up with `new_field_title` in update params.

Reply:
I might be misunderstanding you, but it looks right to me. The return hash in '_create_field/returns_a_field_hash.yml' gives the title string as "Custom text" and the part you've quoted in '_update_field/returns_a_user_hash.yml' shows the return hash from the update has the title string as "Custom text - updated".
      
```
# spec/cassettes/ActiveCampaign_API_Fields/_create_field/returns_a_field_hash.yml
# ln 8
string: '{"field":{"title":"Custom text","type":"text","descript":"A custom
```

Isn't that correct?

I don't think the overriding of `let(:field_title)` at 'spec/lib/active_campaign/api/fields_spec.rb:39' should affect the value anywhere else. My understanding, anyway, is that they're all recreated from scratch for each example to avoid changes in one from affecting another.

Reply:
If you have a look at the cassette `_create_field/returns_a_field_hash.yml` is not used for `_update_field/returns_a_field_hash.yml`. The whole thing with the `let` is that you can override the ones for the context if you want. 

Since we want to test the update (not the create) we do not want to update them. See my next comment